src/main/java/database/Database.csv

```diff
@@ -0,0 +1,22 @@
+Apple,20.0,1000,[],Mon Nov 28 19:17:30 EST 2022,
```

Comment:
You should consider documenting the format that the database would follow in a section in the readme while, and make reference to this in the classes that operate on the database